src/core/dbus-execute.c

```diff
@@ -933,7 +933,7 @@ const sd_bus_vtable bus_exec_vtable[] = {
         SD_BUS_PROPERTY("NoNewPrivileges", "b", bus_property_get_bool, offsetof(ExecContext, no_new_privileges), SD_BUS_VTABLE_PROPERTY_CONST),
         SD_BUS_PROPERTY("SystemCallFilter", "(bas)", property_get_syscall_filter, 0, SD_BUS_VTABLE_PROPERTY_CONST),
         SD_BUS_PROPERTY("SystemCallArchitectures", "as", property_get_syscall_archs, 0, SD_BUS_VTABLE_PROPERTY_CONST),
-        SD_BUS_PROPERTY("SystemCallErrorNumber", "i", bus_property_get_int, offsetof(ExecContext, syscall_errno), SD_BUS_VTABLE_PROPERTY_CONST),
+        SD_BUS_PROPERTY("SystemCallErrorAction", "i", bus_property_get_int, offsetof(ExecContext, syscall_error_action), SD_BUS_VTABLE_PROPERTY_CONST),
```

Comment:
na, we can't do it like that. this is API, we need to stick to it. breaking it is only OK with very strong cases for it, and i don't think this qualifies.

if we rename props, we need to keep them available under their old names too for compat, and maybe hide them with SD_BUS_VTABLE_PROPERTY_HIDDEN or so

Reply:
That said, I am not sure it's really worth renaming this at all. yes, "Number" is a bit of a misnomer, but it's not drastically borked, just a bit skewed. I#d just stick to the current naming.

Reply:
What about the variable `syscall_errno`? The commit would be greatly simplified if also that name would be retained.

Reply:
yes, retain it

Reply:
fixed